src/quest_manager/templates/quest_manager/preview_content_submissions.html

```diff
@@ -30,7 +30,22 @@
   </div>
   {{s.quest.submission_details|safe}}
 </li>
-<li class="list-group-item list-group-item-info">Submission Comments</li>
-{% with s.get_comments as target_comments %}
-  {% include "comments/comments.html" %}
-{% endwith %}
+{% if s.is_returned or completed %}
+  <li class="list-group-item list-group-item-info">Submission Comments</li>
+  {% with s.get_comments as target_comments %}
+    {% include "comments/comments.html" %}
+  {% endwith %}
+{% endif %}
+<!-- will load a student quick reply form for submissions that have been returned or handed in this course -->
+{% if s.is_returned or completed and not past %}
```

Comment:
We should include a quick reply on past quests.  So you can remove this logic and just go with the one overarching if statement higher up.

Reply:
"not past" clause removed on all associated if statements and redundant statement removed.